features/configuration/backtrace_exclusion_patterns.feature

```diff
@@ -1,7 +1,19 @@
 Feature: Excluding lines from the backtrace
 
-  To reduce the noise when diagnosing failures, RSpec excludes matching lines
-  from backtraces. The default exclusion patterns are:
+  To reduce the noise when diagnosing failures, RSpec can exclude lines
+  belonging to certain gems or matching given patterns.
+
+  If you want to filter out backtrace lines belonging to specific gems, you can
+  use `config.filter_gems_from_backtrace` like so:
+
+  ```
+  config.filter_gems_from_backtrace "ignored_gem", "another_ignored_gem",
```

Comment:
Is it worth wrapping this in `RSpec.configure do |config|...end` to show the full context? As it exits, it's barely different from what's in the paragraph above.